aries-site/src/examples/templates/forms/ShippingExample.js

```diff
@@ -2,129 +2,31 @@ import React, { useContext } from 'react';
 import {
   Box,
   Button,
-  CheckBox,
   Form,
-  FormField,
   Header,
   Heading,
-  MaskedInput,
   ResponsiveContext,
-  Select,
   Text,
-  TextInput,
 } from 'grommet';
+import { ShippingInfomation, ContactInformation } from './Shipping';
 
-const emailMask = [
-  {
-    regexp: /^[\w\-_.]+$/,
-    placeholder: 'jane.smith',
-  },
-  { fixed: '@' },
-  {
-    regexp: /^[\w]+$/,
-    placeholder: 'hpe',
-  },
-  { fixed: '.' },
-  {
-    regexp: /^[\w]+$/,
-    placeholder: 'com',
-  },
-];
-
-const states = [
-  'AK',
-  'AL',
-  'AR',
-  'AZ',
-  'CA',
-  'CO',
-  'CT',
-  'DC',
-  'DE',
-  'FL',
-  'GA',
-  'HI',
-  'IA',
-  'ID',
-  'IL',
-  'IN',
-  'KS',
-  'KY',
-  'LA',
-  'MA',
-  'ME',
-  'MD',
-  'MN',
-  'MO',
-  'MS',
-  'MT',
-  'NC',
-  'ND',
-  'NE',
-  'NH',
-  'NJ',
-  'NM',
-  'NY',
-  'NV',
-  'OH',
-  'OK',
-  'OR',
-  'PA',
-  'RI',
-  'SC',
-  'SD',
-  'TN',
-  'TX',
-  'UT',
-  'VA',
-  'VT',
-  'WA',
-  'WI',
-  'WV',
-  'WY',
-];
-
-const phoneMask = [
-  { fixed: '(' },
-  {
-    length: 3,
-    regexp: /^[0-9]{1,3}$/,
-    placeholder: 'XXX',
-  },
-  { fixed: ')' },
-  { fixed: ' ' },
-  {
-    length: 3,
-    regexp: /^[0-9]{1,3}$/,
-    placeholder: 'XXX',
-  },
-  { fixed: '-' },
-  {
-    length: 4,
-    regexp: /^[0-9]{1,4}$/,
-    placeholder: 'XXXX',
-  },
-];
-
-const emailValidation = [
-  {
-    regexp: new RegExp('[^@ \\t\\r\\n]+@'),
-    message: 'Enter a valid email address.',
-    status: 'error',
-  },
-  {
-    regexp: new RegExp('[^@ \\t\\r\\n]+@[^@ \\t\\r\\n]+\\.[^@ \\t\\r\\n]+'),
-    message: 'Enter a valid email address.',
-    status: 'error',
-  },
-  {
-    regexp: new RegExp('[^@ \\t\\r\\n]+@[^@ \\t\\r\\n]+\\.[^@ \\t\\r\\n]+'),
-    message: 'Enter a valid email address.',
-    status: 'error',
-  },
-];
+export const ShippingFormExample = () => {
```

Comment:
So I broke this up in order to just pass in the `children` aka the 
```
      <ShippingInfomation />
      <ContactInformation />
```

that way I didn't have to use the same `state, onSubmit, onValidate code`

Reply:
I'm still not following... 😬 I'm not understanding where we would be "using the same." Can you point me to where that happens?

Why don't we just put them directly into ShippingExample? Then we could eliminate ShippingFormExample.

```
<Box>
  <Header ...>
    ...
  </Header>
  <Box ...>
    <Form ...>
      <ShippingInformation />
      <ContactInformation />
      <Box ...>
        <Button label="Continue" primary type="submit" />
      </Box>
    </Form>
  </Box>
</Box>
```

Reply:
I updated the examples let me know if they make better sense. 
I did have to duplicate this portion of the code since it needed unique `name` to set the focus 
`const targetFormField = document.getElementsByName(target);`

```
  const [formValues, setFormValues] = React.useState({
    'address1': '',
    'address2': '',
    'city': '',
    'state': '',
    'zipcode': '',
    'isBusiness': '',
    'full-name-1': '',
    'phone-shipping-1': '',
    'email-shipping-1': '',
  });

  // eslint-disable-next-line no-unused-vars
  const onSubmit = ({ value, touched }) => {
    // Your submission logic here
  };

  // provide order of formfields for validation
  // to properly place focus on any errors or infos
  const formFields = [
    'address1',
    'address2',
    'city',
    'state',
    'zipcode',
    'isBusiness',
    'full-name-1',
    'phone-shipping-1',
    'email-shipping-1',
  ];

  // On long forms, we want to focus the first of any fields that
  // return an error or info message. This removes the need for the
  // user to scroll to find which field blocked form submission.
  const onValidate = validationResults => {
    const target = formFields.find(
      field =>
        field in validationResults.errors || field in validationResults.infos,
    );
    if (target) {
      const targetFormField = document.getElementsByName(target);
      targetFormField[0].focus();
    }
  };
```